### PR TITLE
Add Editing Documentation page

### PR DESCRIPTION
**Pull request by manez**
## Purpose / why

Adds a new page walking through how to edit the documentation using the GitHub in-browser GUI, including the soon-to-be option to directly commit small fixes. 

## What changes were made?

One new page, one new menu entry so it'll be linked in the docs, and five new screenshots.

## Verification

mkdocs build and review. This PR is _not yet ready for merge_ because it is written for a world where some upcoming changes to loosen the workflow on this repo have been implemented. Once those changes are real, this can be merged. 

## Interested Parties

@rosiel @dannylamb @eldonquijote does this seem in line with what was agreed for the new workflow? 

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Does this PR link to related [issues](https://github.com/Islandora/documentation/issues/)?
* [ ] Does the proposed documentation align with the [Islandora Documentation Style Guide](https://islandora.github.io/documentation/contributing/docs_style_guide/)?
* [ ] Are the changes accurate, useful, free of typos, etc?

> __Person merging__ should ensure the following
* [ ] Does mkdocs still build successfully? (This is indicated by TravisCI passing. To test locally, and see warnings, see [How To Build Documentation](https://islandora.github.io/documentation/technical-documentation/docs-build/).)
* [ ] If pages are renamed or removed, have all internal links to those pages been fixed?
* [ ] If pages are added, have they been linked to or placed in the menu?
* [ ] Did the PR receive at least one approval from a committer, and all issues raised have been addressed?

### CODE REVIEW

**Comment by manez** [comment]
I meant to put this out there as a Draft pull request, but I messed up and it doesn't seem to be an option after-the-fact. **DO NOT MERGE**. This documentation _lies_, until some other stuff goes through.

File: docs/contributing/editing-docs.md
Comment (rosiel) [inline docs/contributing/editing-docs.md]: technical question - I don't think you'll be able to make a new branch if you're not part of the islandora github org (i.e. with write access to the repo)? 

Here's the results of trying to edit a file in islandora_ocr, which i don't have write access to: a note at the top of the page that they've created me a fork, and the button says "propose changes". 

Screen Shot 2020-10-26 at 11 50 12 AM
Screen Shot 2020-10-26 at 11 50 38 AM

Comment (manez) [reply docs/contributing/editing-docs.md]: I hadn't considered that. I guess the choose your own adventure needs one more branch. I'll add that in.

Comment (rosiel) [reply docs/contributing/editing-docs.md]: It's not so much a "choose your own adventure" - while we see it that way, someone new to this would not. 

Maybe assume they're "a general person" and talk about "proposing changes" first, yadda yadda github fork. 

If you're a member of the Islandora github organization, the options will be to 'create a branch and make a PR" or to merge directly. In general, do not merge directly into main. Create a PR. 

[image of "pull request" selected. Is there a way to make that the default?]

Pull Request Policy: (does it make sense to separate out the "review" from "merge" here?) PRs must be approved before they are merged. Any [????] can approve a PR. However, PRs can be merged only by someone who is not in the same organization as the author.  (the weird thing here is if it's at all collaborative, it would prevent people who are involved from being able to take action. like recently, i merged a PR of yours @manez that i'd had a commit in.)

(Only then, describe the exception)
PR versus Merge directly:  If your change is _extremely_ minor: a formatting or spelling correction that does not change the semantics of the document, you can skip the PR process and merge directly into the main branch. This option is available to those with write access, i.e. members of the islandora github org, use your power responsibly yadda yadda.

Comment (manez) [reply docs/contributing/editing-docs.md]: I rewrote the "steps" the include what happens if you don't have write access and pushed the "exception" into a note block at the end. 

And added another clause under "Before you start" to link out to some of the GitHub terms we're throwing around.

**Comment by manez** [comment]
I think this should be paired with a "how to create documentation" page, and there's enough difference (editing the index, adding new assets, how to start a new .md file in the GUI) for them to be seperate, but I'm not wedded to having them be two different pages.

**Comment by rosiel** [comment]
Yeah there could maybe be a Documentation section... how's this for a start? 
- How to contribute documentation (the intro to this, also mention that it's in Mkdocs flavoured markdown, follow our style guide (link to page), that we use pull requests (link to new Docs PR workflow page), code of conduct (link to page)).
- Tutorial: Edit Documentation using the Github GUI (most of this page)
- Documentation Style Guide
- Documentation Structure (where stuff lives, how to edit menus, how to add pictures, etc. Maybe also mention 'this repo has the entire islandora history and so if you clone it locally it's big')
- Documentation Pull Request Workflow (/Policy?) (describe who is allowed to do what and when and the pr template etc. This kind of stuff, the Policy stuff, seems like it could get its own page instead of being part of a tutorial or something? (do you need a CLA to contribute a PR? This is not clear from contributing.)
- Tutorial: Build and serve the documentation locally with Mkdocs

This would be a bit of a resturcturing - moving existing pages into a 'documenation' section/folder

**Comment by manez** [comment]
Oof. You do indeed need a CLA to make a PR. I'll stick that in the "Before you start" of this draft for now, but I like the approach you've outlined for a more comprehensive documentation section.

**Review by rosiel — APPROVED** [review]
So, if this is the policy, then I can't merge, because I left some wording suggestions. But I like it!

**Comment by manez** [comment]
It needs to sit until policy & settings get updated anyhow, since the process described here is not yet real.

For policy, I'm not sure. I wouldn't consider accepted suggestions to be a contribution barring a merge, since there's no commit, but I guess I did copy and paste some of your wording directly. I thought saying "anyone who contributed a commit" might be too opaque for new users who aren't marinated in GitHub concepts, but if "collaborated with you on writing the changes" is misleading, maybe it needs the more technical explanation.

**Comment by manez** [comment]
Thanks @eldonquijote and @rosiel ! This "General Member" merge demonstrates that the instructions in this doc are now true 😄

**Comment by rosiel** [comment]
Thank you @eldonquijote !!

**Pull request merged**
